src/main/java/com/netchill/webservices/api/MovieWs.java

```diff
@@ -31,4 +31,15 @@ public Production getMovieById(@PathParam("id") Long id) {
         return this.movieService.getMovieById(id);
     }
 
+    @GET
+    @Path("/top-rated")
+    public List<Production> getTopRated() {
+        return movieService.getTopRated();
+    }
+
+    @GET
+    @Path("/top-rated")
+    public List<Production> getTopRatedByGenre(@QueryParam("genre") int genre) {
+        return movieService.getTopRatedByGenre(genre);
+    }
```

Comment:
il faut vérifier avec postman si les 2 endpoints sont accessible.
Si ce n'est pas le cas, il faudrais en faire 1 seul :

```suggestion
    @GET
    @Path("/top-rated")
    public List<Production> getTopRatedByGenre(@QueryParam("genre") Integer genre) {
        if(genre == null) {
            return movieService.getTopRated();
        }
        
        return movieService.getTopRatedByGenre(genre);
    }
```

Reply:
idem tu ne m'as pas répondu

Reply:
Quand j'ai voulu debug l'appli ça me disait que y'avait une ambiguïté ducoup j'ai changé.